tests/kafkatest/services/security/security_config.py

```diff
@@ -166,10 +167,22 @@ def has_sasl_kerberos(self):
 
     @property
     def kafka_opts(self):
+        if SecurityConfig.ZK_JAAS_LOGIN:
+            return "\"-Djava.security.auth.login.config=%s\"" % SecurityConfig.ZK_JAAS_LOGIN
```

Comment:
I agree. It might make your task simpler to make `self.has_sasl` a property (like `sasl_mechanism`, `security_protocol`, etc)